articles/postgresql/flexible-server/concepts-networking-private.md

```diff
@@ -118,10 +118,10 @@ The custom DNS server should be inside the virtual network or reachable via the
 
 ### Private DNS zone and virtual network peering
 
-Private DNS zone settings and virtual network peering are independent of each other. If you want to connect to the Azure Database for PostgreSQL flexible server from a client that's provisioned in another virtual network from the same region or a different region, you have to *link* the Private DNS zone with the virtual network. For more information, see [Link the virtual network](/azure/dns/private-dns-getstarted-portal#link-the-virtual-network).
+Private DNS zone settings and virtual network peering are independent of each other. If you want to connect to the Azure Database for PostgreSQL Flexible Server from a client that's provisioned in another virtual network from the same region or a different region, you have to *link* the Private DNS zone with the virtual network. For more information, see [Link the virtual network](/azure/dns/private-dns-getstarted-portal#link-the-virtual-network).
 
 > [!NOTE]
-> Only Private DNS zone names that end with `postgres.database.azure.com` can be linked. Your DNS zone name can't be the same as your Azure Database for PostgreSQL flexible servers. Otherwise, name resolution fails.
+> Only Private DNS zone names that end with `postgres.database.azure.com` can be linked. Your DNS zone name can't be the same as your Azure Database for PostgreSQL Flexible Servers. Otherwise, name resolution fails.
```

Comment:
```suggestion
> Only Private DNS zone names that end with `postgres.database.azure.com` can be linked. Your DNS zone name can't be the same as your Azure Database for PostgreSQL flexible servers. Otherwise, name resolution fails.
```